libs/category/state/src/reducers/category/category.reducer.ts

```diff
@@ -54,6 +54,7 @@ export function daffCategoryReducer<U extends DaffGenericCategory<U>, W extends
         categoryPageMetadata: {
           ...initialState.categoryPageMetadata,
           ...buildMetadataFromRequest(action.request),
+          id: action.request.id,
```

Comment:
Should we even set this upon load, or only upon load success?

Reply:
How much of the requests do we want to optimistically apply to state? I thought everything but maybe it should only be the filters.

Reply:
bump

Reply:
In retrospect, I'm not sure it matters.